roles/openshift_node_bootstrap_configmap/templates/node-config.yaml.j2

```diff
@@ -0,0 +1,53 @@
+allowDisabledDocker: false
+apiVersion: v1
+authConfig:
+  authenticationCacheSize: 1000
+  authenticationCacheTTL: 5m
+  authorizationCacheSize: 1000
+  authorizationCacheTTL: 5m
+dnsBindAddress: "127.0.0.1:53"
+dnsDomain: cluster.local
+dnsIP: 0.0.0.0
+dnsNameservers: null
+dnsRecursiveResolvConf: /etc/origin/node/resolv.conf
+dockerConfig:
+  dockerShimRootDirectory: /var/lib/dockershim
+  dockerShimSocket: /var/run/dockershim.sock
+  execHandlerName: native
+enableUnidling: true
+imageConfig:
+  format: "{{ openshift_imageconfig_format }}"
+  latest: false
+iptablesSyncPeriod: 30s
+kind: NodeConfig
+kubeletArguments:
+  cloud-config:
+  - /etc/origin/cloudprovider/{{ openshift_node_bootstrap_configmap_cloud_provider }}.conf
+  cloud-provider:
+  - {{ openshift_node_bootstrap_configmap_cloud_provider }}
+  node-labels: {{ openshift_node_bootstrap_configmap_default_labels | union(openshift_node_bootstrap_configmap_custom_labels) | list | to_json }}
+masterClientConnectionOverrides:
+  acceptContentTypes: application/vnd.kubernetes.protobuf,application/json
+  burst: 40
+  contentType: application/vnd.kubernetes.protobuf
+  qps: 20
+masterKubeConfig: node.kubeconfig
+networkConfig:
+  mtu: "{{ openshift_node_bootstrap_configmap_network_mtu }}"
```

Comment:
https://github.com/openshift/openshift-ansible/pull/5972 does similar except it uses `openshift_node_sdn_mtu` which is the customer facing value today.  Can update here as needed.

Reply:
@smarterclayton, this does as well but I transfer that value into the role version here: https://github.com/openshift/openshift-ansible/pull/5923/files#diff-e8f255754ee97259ae8e087cba95130aR15